qt_ui/windows/AirWingConfigurationDialog.py

```diff
@@ -823,6 +823,8 @@ def save_config(self) -> None:
         )
         fd.setAcceptMode(QFileDialog.AcceptMode.AcceptSave)
         if fd.exec_():
+            for tab in self.tabs:
+                tab.apply()
```

Comment:
Could you elaborate this change? It might just be me, but I can't seem to understand how this applies to the rest of the code 😅

Reply:
Currently we don't have a way to access the updated squadron data, we do that intentionally so that a user can revert their changes if they want. Because of that, when you save it would save squadrons even if they were deleted in the UI. To fix it I just called the apply method so the squadron gets updated when a user saves the faction. I also have since added a dialog box to warn the user that they can't revert changes after saving a squadron.